apps/explorer/src/components/validator/calculateAPY.ts

```diff
@@ -3,6 +3,8 @@
 
 import type { ActiveValidator } from '~/pages/validator/ValidatorDataTypes';
 
+const ROUNDING_MODE = 4;
```

Comment:
Not blocking but probably would call this something more useful like `APY_DECIMALS`